python/sglang/srt/managers/tokenizer_manager.py

```diff
@@ -856,8 +856,8 @@ async def update_weights_from_distributed(
     ) -> Tuple[bool, str]:
         self.auto_create_handle_loop()
         assert (
-            self.server_args.dp_size == 1
-        ), "dp_size must be for update weights from distributed"
+            self.server_args.dp_size == 1 or self.server_args.enable_dp_attention
+        ), "dp_size must be 1 or dp attention is enabled for update weights from distributed"
```

Comment:
nit: dp_size must be 1 or dp attention `must be` enabled

Reply:
fixed.